### ISSUE TITLE
[Feature Request] Custom Volume Booster something

### ISSUE THREAD

**Issue by Brian2po** [Lyall/MetaphorFix#108]
My Game volume is too low even setting computer volume at Max, can't hear Dialogue etc.

**Comment by Tekno560** [comment]
Do you have the volume maxed in windows settings too? This isn't a problem at all for me on my laptop or my desktop

**Comment by Lyall** [comment]
The latest game updated boosted the max volume slider to 6 times higher than before. Might be worth checking the volume mixer in windows as Tekno560 said. Do you have any audio issues in other games?

**Comment by system2ckc** [comment]
I wonder if some or all anime cutscenes are not boosted along with the master volume change they made though. I did have that happen for atleast one anime cutscene but i did not check if it was prevalent in all of them. I actually personally lowered my master volume back to 17-25 range just to make sure there's volume consistency. OP needs to be clear if they are referring to the game in general being low volume or just the anime cutscenes.

**Comment by ssj3rd** [comment]
Yeah, the anime videos are too quiet, you can barely understand the dialog. In the rest of the game the audio is just fine.

**Comment by bluekaiser07** [comment]
I can barely hear anything. The game's volume is maxed both in PC settings and ingame volume settings.

**Comment by Lyall** [comment]
Not sure what the options would be here. The way that MetaphorFix used to increase the volume is the same place that the official game update fixes it. If you're using any "stereo enhancements" or applying any kind of virtual surround, I would suggest experimenting with disabling that.
